storage_service/locations/tests/test_package.py

```diff
@@ -422,6 +451,12 @@ def test_replicate_aip_twice(self):
         assert aip.replicas.count() == 2
         for replica in aip.replicas.all():
             assert replica.replicas.count() == 0
+            repl_dir = ""
+            for repl_path in (replication_dir, replication_dir2):
+                if replica.current_location.full_path.startswith(repl_path):
+                    repl_dir = repl_path
+                    break
+            self._test_bagit_structure(replica, replica.uuid, repl_dir)
```

Comment:
I think that our tests would be more readable at a glance if we avoided loops.
That may not always be the case, but in this particular scenario what do you think about this alternative? I haven't tested, it may require some tweaking.

```python
assert aip.replicas.count() == 2
assert aip.replicas[0].replicas.count() == 0
assert aip.replicas[1].replicas.count() == 0
self._test_bagit_structure(
    replica[0], aip.current_location.replicators[0].relative_path)
self._test_bagit_structure(
    replica[1], aip.current_location.replicators[1].relative_path)
```

Reply:
Yeah, I tried to avoid them, but we can't access these objects through indices. I've proposed a solution that works, and I think might fall nicely in-between. Let me know.